vulnerable_people_form/tier_lookup/lacode_tier_loookup.py

```diff
@@ -0,0 +1,51 @@
+import yaml
+
+from datetime import datetime
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+LOCAL_RESTRICTIONS_FILE = "local-restrictions.yaml"
+PATH_TO_RESTRICTIONS_FILE = f"vulnerable_people_form/tier_lookup/{LOCAL_RESTRICTIONS_FILE}"
+COLLECTIONS_GIT_REPO = "https://github.com/alphagov/collections.git"
+DEFAULT_NATIONAL_ALERT_LEVEL = 3
+
+
+class LacodeTierLookup:
+    """
+    fetches the tier status for supplied LA Code
+    """
+
+    def __init__(self,
+                 tier_file_path: str = None):
+        self.tier_file_path = tier_file_path
+
+    def lacode_to_tier(self, lacode: str, date: datetime = datetime.now()):
```

Comment:
I think we need to be careful with what the timezone is. I suspect that apps on PaaS are in UTC whereas here we need to make sure we're using whatever the timezone in the UK is.

I think this is true for all the interpretation of the datetimes in the YAML—they all should be interpreted as local UK times.